b2sdk/exception.py

```diff
@@ -362,6 +366,16 @@ class NonExistentBucket(B2SimpleError):
     prefix = 'No such bucket'
 
 
+class FileOrBucketNotFound(B2SimpleError):
```

Comment:
Maybe? I think it'd be backward-compatible
```
      ResourceNotFound
            |
    FileOrBucketNotFound
    /                  \
NonExistentBucket    FileNotPresent
```

Reply:
Seems reasonable to me.